### PR TITLE
fix(changelog): display summary of merged PRs at top of changelog.  Clos...

### PR DESCRIPTION
**Pull request by sdheisenberg**
...es #1416

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

**Comment by bacongobbler** [comment]
Looks great! I did something similar with confd's changelog (merges vs commits). I'm wondering if we should make this the default instead. @mboersma, what's your opinion on this?

**Comment by mboersma** [comment]
I like this addition:

```console
$ ./contrib/util/generate-changelog.sh v1.5.0
### v1.5.0 -> HEAD

####Merged pull Requests:
02c6337 Merge branch 'issue-1416' of https://github.com/sdheisenberg/deis into sdheisenberg-issue-1416
54757a0 Merge pull request #3498 from carmstrong/organize-contribs
6fe16b1 Merge pull request #3504 from mboersma/update-aws-docs
b48ced3 Merge pull request #3506 from crigor/docs-troubleshooting-ssh
fe799ae Merge pull request #3499 from bacongobbler/3475-add-unittest
c13170f Merge pull request #3487 from econnell/provisioner-error-checking
ce21c3b Merge pull request #3484 from pmelmon/master

#### Features

 - [`ecc1acc`](https://github.com/deis/deis/commit/ecc1acc21e16a55b8bf1d1b6ab641134af309ea6) controller: Add users:list
 - [`34a2289`](https://github.com/deis/deis/commit/34a2289391bc2b61734575d098273730d6ff3fd1) contrib/azure/azure-coreos-cluster: add an https enddpoint on creation
```

**Comment by carmstrong** [comment]
Looks good to me too. Code LGTM.

**Comment by mboersma** [comment]
Code LGTM.

File: contrib/util/generate-changelog.sh [hunk 1]
```diff
@@ -36,6 +36,13 @@ subheading() {
     echo
 }
 
+summary() {
+  # show the merged pull requests
+  printf "####Merged pull Requests:\n"
```
Comment (mboersma) [inline contrib/util/generate-changelog.sh]: Should be a space here, after `####`.

**Comment by bacongobbler** [comment]
Hmm, how about this for formatting?

```
#### Merged Pull Requests:

 - a8d4f99 chore(contrib/coreos): upgrade fleet to 0.9.2
 - 22179b9 docs(logger): change incorrect ref to db component
 - 64ebf58 fix(registry): create /deis/cache etcdctl dir
 - 07e58f9 ref(tests): move route53-wildcard.py script to contrib/ec2
 - 88c8f54 docs(faq): summarize difference between deis and deisctl clients
 - 2264a0d fix(deisctl): omit cache from `deisctl stop platform"
 - 78e4998 Proposal: SSL support for custom domains
 - 7367202 Proposal: deis client plugins
```

This is generated using `git --no-pager log --merges --format=" - %h %b" $FROM..$TO`

**Comment by mboersma** [comment]
I prefer the approach in #3520, wherein we simply use the merges for all sections of the CHANGELOG.md. (Honestly after you pointed it out @sdheisenberg, I'm surprised we haven't been doing this all along. It's a much clearer summary of work done on Deis.)

:100: for the work done in this PR, thanks again!

**Pull request closed without merging**